book/accessibility.md

```diff
@@ -1064,33 +1059,23 @@ class Tab:
 
 To draw an outline, we'll use `DrawOutline`:
```

Comment:
I don't see DrawOutline in the code below any more, this line needs to be adjusted.

Reply:
Yes, because we now introduce it in Ch 7 to draw the browser chrome!